src/styles/brackets.less

```diff
@@ -220,8 +220,8 @@ a, img {
 
     #status-language {
         border-right: 1px solid @bc-panel-border;
-        padding: 0px;
```

Comment:
There’s a practice to leave off the unit from zero length values. It makes the code look slightly cleaner.